src/pages/iv-opening-qty/forms/InventoryOpeningQtysForm.js

```diff
@@ -0,0 +1,223 @@
+import { Checkbox, fabClasses, FormControlLabel, Grid } from '@mui/material'
+import * as yup from 'yup'
+import toast from 'react-hot-toast'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import FormShell from 'src/components/Shared/FormShell'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { useContext, useEffect } from 'react'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { useInvalidate } from 'src/hooks/resource'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { useForm } from 'src/hooks/form'
+import { InventoryRepository } from 'src/repositories/InventoryRepository'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import CustomTextArea from 'src/components/Inputs/CustomTextArea'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { ControlContext } from 'src/providers/ControlContext'
+import { ResourceLookup } from 'src/components/Shared/ResourceLookup'
+import CustomNumberField from 'src/components/Inputs/CustomNumberField'
+
+const InventoryOpeningQtysForm = ({ labels, maxAccess, recordId, record }) => {
+  const { postRequest, getRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+
+  const invalidate = useInvalidate({
+    endpointId: InventoryRepository.InventoryOpeningQtys.qry
+  })
+
+  const { formik } = useForm({
+    maxAccess,
+    initialValues: {
+      recordId: recordId || null,
+      year: '',
+      siteId: '',
+      sku: '',
+      itemId: '',
+      itemName: '',
+      qty: '',
+      pieces: '',
+      avgWeight: '',
+      lotCategoryId: '',
+      trackBy: '',
+      pieces: ''
+    },
+    enableReinitialize: true,
+    validateOnChange: true,
+    validationSchema: yup.object({
+      year: yup.string().required(),
+      siteId: yup.string().required(),
+      sku: yup.string().required(),
+      itemId: yup.string().required(),
+      qty: yup.number().required().min(0.01, 'Quantity must be greater than 0'),
```

Comment:
remove the messages because of translation

Reply:
its hidden